src/platyPS/platyPS.psm1

```diff
@@ -599,6 +603,70 @@ function New-MarkdownAboutHelp
     }
 }
 
+function New-YamlHelp
+{
+    [CmdletBinding()]
+    [OutputType([System.IO.FileInfo[]])]
+    param(
+        [Parameter(Mandatory=$true,
+            Position=1,
+            ValueFromPipeline=$true,
+            ValueFromPipelineByPropertyName=$true)]
+        [string[]]$Path,
+
+        [Parameter(Mandatory=$true)]
+        [string]$OutputPath,
```

Comment:
If you want ot use `$OutputPath` and not `$OutputFolder`, please add handling for both paths and folders cases. For example, see how `New-ExternalHelp` allows to pass both dir and `*.xml` file as `$OutputPath` parameter